src/Altinn.FileScan/Services/DataElementService.cs

```diff
@@ -48,7 +48,7 @@ public async Task Scan(DataElementScanRequest scanRequest)
 
                 var stream = await _repository.GetBlob(scanRequest.Org, scanRequest.BlobStoragePath, scanRequest.StorageContainerNumber);
 
-                var filename = string.IsNullOrEmpty(scanRequest.Filename) ? $"{scanRequest.DataElementId}.txt" : scanRequest.Filename;
+                var filename = $"{scanRequest.DataElementId}.txt";
```

Comment:
What about using .bin instead?
```suggestion
                var filename = $"{scanRequest.DataElementId}.bin";
```

Reply:
Done